src/com/nutrons/framework/util/FlowOperators.java

```diff
@@ -5,6 +5,8 @@
 import java.util.concurrent.TimeUnit;
 import java.util.function.Supplier;
 
+import static java.lang.Math.abs;
+
 public class FlowOperators {
   /**
    * Generate a Flowable from a periodic call to a Supplier.
```

Comment:
Not the purpose of this pull request, maybe in another branch will do
I'm not sure why the time was set as such and so I don't think I should edit it